charts/graylog/values.yaml

```diff
@@ -473,7 +473,9 @@ graylog:
     ## Init Container image
     ##
     image:
+      registry: "docker.io"
       repository: "alpine"
```

Comment:
You can just specify the version here e..g `alpine:3.17.0`

Reply:
@KongZ I think separated key for tag is better